client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-info-panel/incident-info-panel.component.ts

```diff
@@ -57,7 +57,7 @@ export class IncidentInfoPanel implements AfterViewInit {
 
     this.router.queryParams.subscribe((params: ParamMap) => {
       if(params['preview'])
```

Comment:
params['preview'] is returning a boolean I beleive, so you can probably just do `this.showWarning = params['preview']`